tests/phpunit/tests/st/strategy/shortcode/test-wpml-pb-register-shortcodes.php

```diff
@@ -41,9 +42,6 @@ public function register_shortcode_strings( $encode ) {
 			'args'  => array( $post_id, WPML_PB_Integration::MIGRATION_DONE_POST_META, true ),
 		) );
 
-
-		$dummy_content = rand_long_str( 100 ) . $shortcode . rand_long_str( 100 );
```

Comment:
I removed the content before and after, because now it would be wrapped and we would register 2 more strings. This case is considered in the next test: `it_should_register_strings_in_non_wrapped_content`.